app/scripts/controllers/inventory/viewStockLevels.js

```diff
@@ -0,0 +1,338 @@
+'use strict';
+
+angular.module('bsis')
+  .controller('ViewStockLevelsCtrl', function($scope, $location, $filter) {
+
+    var master = {
+      inventoryStatus: 'IN_STOCK',
+      allSites: true
+    };
```

Comment:
@timler Please initialise the distribution center to null here. It makes it easier to see what variables are on the scope then.